app/graphql/resolvers/rdf.ts

```diff
@@ -314,6 +322,107 @@ export const dataCubesMetadata: NonNullable<
   );
 };
 
+export const dataCubesObservations: NonNullable<
+  QueryResolvers["dataCubesObservations"]
+> = async (_, { locale, filters }, { setup }, info) => {
```

Comment:
It seems like there are quite a few places where this function can throw an error.  I'm not sure what the suspense boundary looks like above this, but maybe these errors could be caught within this function and returned a little more gracefully?  Unless graphQL does a good job at handeling all these thrown errors?

Reply:
When an error is thrown here, like the `"Can't query multiple cubes observations without joinBy!"`, we would see it in an error banner in the place of the chart with this exact message. I feel like handling this _correctly_ would be a part of bigger change as discussed on Wednesday; I think I'd prefer to first have a clear picture in mind how to handle such errors in general (in this case, server-side errors) if that makes sense 😄